pkgs/native_toolchain_c/test/clinker/objects_cross_test.dart

```diff
@@ -2,31 +2,31 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-//TODO(mosuem): Enable for windows and mac.
+// TODO(mosuem): Enable for windows.
 // See https://github.com/dart-lang/native/issues/1376.
-@TestOn('linux')
+@TestOn('linux || mac-os')
```

Comment:
(Hm, they should have made a bunch of const objects and an `Or([Linux(), MacOS()])` class instead of going for strings. How do we know if this syntax is correct and we're not just skipping the test everywhere because of wrong syntax. 😄 )